docs/conf.py

```diff
@@ -61,10 +62,11 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
+import lasagne
 # The short X.Y version.
-version = '0.1'
+version = '.'.join(lasagne.__version__.split('.', 3)[:2]).rstrip('dev')
 # The full version, including alpha/beta/rc tags.
-release = '0.1'
+release = lasagne.__version__
```

Comment:
Why would it work with `virtualenv` but not otherwise? I'm not sure how it's currently set up - there seem to be multiple virtualenv-related settings, some of them are enabled and some aren't. It's a bit unclear what they all do. I remember having to tinker with them for a while before everything would work, though.